### PR TITLE
Update helper that removes plans with invalid jurisdiction_id values

### LINKED ISSUE #1251: RVL-1133 - Monitoring Tab not loading plans
- [ ] The monitoring tab on the Thailand WebUI Preview environment, is not loading plans. See screenshot below:
image

### PR DESCRIPTION
**Pull request by p-netm**
the mentioned helper should now also:
- filter out plans whose jurisdiction_id value is falsy

close #1251

### CODE REVIEW

File: src/helpers/utils.tsx [hunk 1]
```diff
@@ -458,15 +458,15 @@ export function getColor(taskObject: InitialTask): Color {
...
  * @param {Plan []} plansArray - a list of pre-filtered plans
  *
  * @return {Plan []} - A list of filtered plans
  */
 export const removeNullJurisdictionPlans = (plansArray: Plan[]): Plan[] => {
   return plansArray.filter((plan: Plan) => {
-    const jurisdictionID = plan.jurisdiction_id.toLowerCase().trim();
-    return !jurisdictionID.includes('null');
+    const jurisdictionId = plan.jurisdiction_id?.toLowerCase().trim();
+    return !!jurisdictionId && !jurisdictionId.includes('null');
```
Comment (moshthepitt) [inline src/helpers/utils.tsx:469]: Is it completely safe to remove `.trim()`?

Comment (p-netm) [reply src/helpers/utils.tsx:469]: I don't think so. If I remember correctly the values that were originally handled by this looked sth like ~  '  NULL'

Comment (moshthepitt) [reply src/helpers/utils.tsx:469]: yeah I recall that as well.  Should we add trim back to this code and ensure we have a test that covers the `' NULL'` cases?

**Review by ciremusyoka — APPROVED** [review]

File: src/helpers/utils.tsx (hunk 1 above)
Comment (ciremusyoka) [reply src/helpers/utils.tsx:469]: The existing function [test](https://github.com/onaio/reveal-frontend/blob/4b8394d538e2cb9c7fe77e792e32849792ea67ef/src/helpers/tests/utils.test.tsx#L326-L330) covers `'  NULL'`  cases ([fixtures](https://github.com/onaio/reveal-frontend/blob/4b8394d538e2cb9c7fe77e792e32849792ea67ef/src/store/ducks/tests/fixtures.ts#L187-L217)).

**Review by moshthepitt — APPROVED** [review]

**Comment by p-netm** [comment]
@moshthepitt @ciremusyoka This pr seems to have been overriden by a force push, i.e. after it was merged into the parent branch. going to re-open this
